firebase_admin/db.py

```diff
@@ -900,13 +900,13 @@ def __init__(self, credential, base_url, timeout, params=None):
           credential: A Google credential that can be used to authenticate requests.
           base_url: A URL prefix to be added to all outgoing requests. This is typically the
               Firebase Realtime Database URL.
-          timeout: HTTP request timeout in seconds. If not set connections will never
+          timeout: HTTP request timeout in seconds. If set to None connections will never
```

Comment:
nit: the 'which is the default behaviour ...' part is no longer true.

Reply:
```
If set to None connections will never
timeout, which is the default behavior of the underlying requests library.
```

That is still true. The `requests` library behavior hasn't yet changed. The default `timeout=None` behavior still comes from there.

Reply:
Oops; yeah I missed the change from 'if unset' to 'if set to none' (despite suggesting exactly that earlier. 😳)